Content.Client/MassMedia/Ui/NewsWriterMenu.xaml.cs

```diff
@@ -0,0 +1,97 @@
+using Content.Client.TextScreen;
+using Content.Client.UserInterface.Controls;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.XAML;
+using Content.Shared.MassMedia.Systems;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.Timing;
+
+namespace Content.Client.MassMedia.Ui;
+
+[GenerateTypedNameReferences]
+public sealed partial class NewsWriterMenu : FancyWindow
+{
+    private readonly IGameTiming _gameTiming;
```

Comment:
Use dependency attribute and IoCManager.InjectDependencies(this)

Reply:
Aren't we supposed to limit the use of IoCManagert.InjectDependencies?

Reply:
i dont think so? also all windows follow the pattern with InjectDependencies